### DISCUSSION TITLE
articles/some-good-fiction-books

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#163] [Comments]
# articles/some-good-fiction-books

今天（4月23日）是「世界读书日」，联合国教科文组织正式的叫法是「世界图书与版权日」。1616年4月23日塞万提斯与莎士比亚同在这一日辞世，因此为了纪念二位文豪将这一天定为「读书日」。有一种说法是4月23日也是莎士比亚的诞辰，这是不可靠的，仅仅是为了凑莎士比亚逝世日的噱头。

https://hutusi.com/articles/some-good-fiction-books

**Comment by semyin** [comment]
准备看《活着》

**Reply by hutusi** [reply]
这本书看的很沉重~~

**Reply by semyin** [reply]
看完了，中间不止一次泪目，很压抑

**Reply by hutusi** [reply]
是呀，余华小说中的人物都活的很艰难~

**Comment by Macin20** [comment]
《小王子》也不错

**Reply by hutusi** [reply]
这本书我也很喜欢👍
